src/smokeTest/java/uk/gov/hmcts/reform/bulkscanprocessor/WhenApplicationDeployedTest.java

```diff
@@ -29,4 +41,14 @@ public void storage_container_exists() {
             .then()
             .body("containers", not(emptyArray()));
     }
+
+    @Test
+    public void can_retrieve_envelopes() {
```

Comment:
I would suggest instead of doing we have a method in Envelope Service like below

```
 public List<Envelope> getPageableEnvelopes() {
        Pageable pageable = PageRequest.of(0, 1);
        Page<Envelope> envelopes = envelopeRepository.findAll(pageable);
        return envelopes.getContent();
    }
```
And also have a new endpoint in controller which uses this method and finally for smoke we call this method.

This way we don't retrieve everything and also validate system is working fine :)

Reply:
I agree with the pagination, but I'd leave that for a different ticket.